openupgradelib/openupgrade.py

```diff
@@ -1031,13 +1032,16 @@ def migrate(no_version=False, use_env=None, uid=None, context=None):
     logging purposes.
     """
     def wrap(func):
-        def wrapped_function(cr, version):
+        @wraps(func)
+        def wrapped_function(cr, version, no_version=no_version, use_env=use_env,
+                             uid=uid, context=context):
```

Comment:
Before, test where failing. This is what I was referring to by it has been working on my local machine: "it works now". IIRC, by trial and error from my previous PR about env support, this is necessary. What alternative explanation would you draw from: https://travis-ci.org/OCA/openupgradelib/jobs/151125711